JuliaInterface/gap/JuliaInterface.gi

```diff
@@ -140,18 +195,11 @@ InstallGlobalFunction( ImportJuliaModuleIntoGAP,
         fi;
     fi;
 
-    _JULIAINTERFACE_PREPARE_RECORD( name );
-    current_module_rec := Julia.(name);
-    Unbind( current_module_rec._JULIAINTERFACE_NOT_IMPORTED_YET );
-    julia_list_func := JuliaFunction( "get_function_symbols_in_module", "GAPUtils" );
-    function_list := StructuralConvertedFromJulia( julia_list_func( JuliaModule( name ) ) );
-    for i in function_list do
-        current_module_rec.(i) := JuliaFunction( i, name );
-    od;
-    julia_list_func := JuliaFunction( "get_variable_symbols_in_module", "GAPUtils" );
-    variable_list := StructuralConvertedFromJulia( julia_list_func( JuliaModule( name ) ) );
-    for i in variable_list do
-        current_module_rec.(i) := JuliaGetGlobalVariable( i, name );
+    current_module := Julia.(name);
+    julia_list_func := JuliaFunction( "get_symbols_in_module", "GAPUtils" );
+    list := StructuralConvertedFromJulia( julia_list_func( current_module!.julia_pointer ) );
+    for i in list do
+        i := current_module.(i);
```

Comment:
I would just call `\.` but not assign the return value.

Reply:
Good idea.